docs/examples/styles/content_align.py

```diff
@@ -0,0 +1,31 @@
+from textual.app import App
+from textual.widgets import Static
+
+
+class ContentAlignApp(App):
```

Comment:
Could you separate this in to Python + CSS. 

I want to discourage people from embedding the CSS because the auto-refresh won't work. You also won't need the `ContentAlignApp` prefix to make it work.

Also, could you avoid using variables. I don't want to introduce too many concepts.

Reply:
I've split them up, and done the same for scrollbar-gutter.